sessionctx/variable/tidb_vars.go

```diff
@@ -584,6 +584,9 @@ const (
 	// TiDBEnableOrderedResultMode indicates if stabilize query results.
 	TiDBEnableOrderedResultMode = "tidb_enable_ordered_result_mode"
 
+	// TiDBEnableOrderbyInSubquery indicates whether to keep ORDER BY in subquery.
+	TiDBEnableOrderbyInSubquery = "tidb_enable_orderby_in_subquery"
```

Comment:
Please add some cases for this variable itself in `set_test.go:TestSetVar(...)`.

Reply:
@qw4990 Updated. Please take another look.